src/components/org-users/views.tsx

```diff
@@ -803,6 +819,17 @@ export function OrganizationUsersPage(
                       ))}
                     </ul>
                   </td>
+                  {props.privileged ? (
+                    <td className="govuk-table__cell">
+                      {props.userExtraInfo[guid].lastLogonTime ? (
+                        format(new Date(props.userExtraInfo[guid].lastLogonTime!), DATE_TIME)
```

Comment:
@kr8n3r This line is a bit dense, doesn't `format()` take an int? and does adding `!` cast to bool?

Reply:
you are correct. seem likr i forgot to update when i was testing outputs from the controller. have updated
as for ! - to stop it complaining about 

```
Argument of type 'number | undefined' is not assignable to parameter of type 'number | Date'.
  Type 'undefined' is not assignable to type 'number | Date'
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Reply:
Aha! I don't feel so bad about not knowing a typescript operator :grin: Thanks for the link!

Reply:
don't beat yourself up to hard :) you were probably think of `!!`